joplin/users/tests/tests.py

```diff
@@ -197,3 +198,21 @@ def test_admin_can_make_departmentless_page(superadmin, rf):
     page_pk = response_json['id']
     created_page = InformationPage.objects.get(id=page_pk)
     assert len(created_page.departments()) is 0
+
+
+@pytest.mark.django_db
+def test_editor_cannot_explore_other_dept_pages(editor, kitchen_service, departmentless_service):
```

Comment:
Do you need to make `editor`, `kitchen_service` and `departmentless_service` into pytest fixtures? I think they should work if you directly import them into test.py. Like what we're doing in other tests: `joplin/pages/service_page/tests.py`.

A pytest fixture only needs to be created in the pytest conftest.py if it is a thing that is only used within pytest and no other part of our app.

Reply:
you're right. I imported instead of putting them in pytest fixtures again, and the tests passed